composer.json

```diff
@@ -30,9 +30,8 @@
     },
     "extra": {
         "branch-alias": {
-            "dev-master": "0.3-dev"
+            "dev-master": "1.1-dev"
```

Comment:
ah, just read #4 - i guess this is the same we did in the other repos, so its fine.